docs/process-services-cloud/process-filter-cloud.component.md

```diff
@@ -0,0 +1,44 @@
+---
+Added: v3.0.0
+Status: Active
+Last reviewed: 2018-21-11
+---
+
+# Process Filter Cloud Component
+
+Lists all available process filters and allows to select a filter.
+
+## Contents
+
+-   [Basic Usage](#basic-usage)
+-   [Class members](#class-members)
+    -   [Properties](#properties)
+    -   [Events](#events)
+
+
+## Basic Usage
+
+```html
+<adf-cloud-process-filters
+    [appName]="currentAppName"
+    [showIcons]="true">
+</adf-cloud-process-filters>
+```
+
+## Class members
+
+### Properties
+
+| Name | Type | Default value | Description |
+| ---- | ---- | ------------- | ----------- |
+| appName | `string` |  | (required) The application name |
+| filterParam | `ProcessInstanceFilterRepresentationModel` |  | (optional) The filter to be selected by default |
+| showIcons | `boolean` | false | (optional) The flag hides/shows icon against each filter |
+
+### Events
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| filterClick | [`EventEmitter`](https://angular.io/api/core/EventEmitter)`<`[`ProcessInstanceFilterRepresentationModel`](../../lib/process-services-cloud/src/lib/process-cloud/models/process-filter-cloud.model.ts)`>` | Emitted when a filter is selected/clicked. |
+| success | [`EventEmitter`](https://angular.io/api/core/EventEmitter)`<any>` | Emitted when filters are loaded successfully. |
+| error | [`EventEmitter`](https://angular.io/api/core/EventEmitter)`<any>` | Emitted when any error occurs while loading the filters. |
```

Comment:
Provide an example of how you can filter the processFilters

### Filtering APS2 task filters

Use the `filterParam` property to restrict the range of filters that are shown:

```html
<adf-cloud-process-filters
   [filterParam]="{name:'Running'}">
</adf-cloud-process-filters>
```

Reply:
the filterParam is used to filter a processFilter from available processFilters